Author: Reiner Herrmann <reiner@reiner-h.de>
Description: change global default settings
 - restrict networking feature; new network namespace can be used to
   circumvent packet filter of default namespace (see #916920).
Bug-Debian: https://bugs.debian.org/916920

--- a/etc/firejail.config
+++ b/etc/firejail.config
@@ -76,7 +76,7 @@
 # networking features should also be enabled (network yes).
 # Restricted networking grants access to --interface, --net=ethXXX and
 # --netfilter only to root user. Regular users are only allowed --net=none.
-# restricted-network no
+restricted-network yes
 
 # Change default netfilter configuration. When using --netfilter option without
 # a file argument, the default filter is hardcoded (see man 1 firejail). This
